lib/rubocop/cop/sevencop.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+require_relative 'sevencop/order_field'
+require_relative 'sevencop/redundant_existence_check'
```

Comment:
For example, executing a task like `bundle exec rake 'new_cop[Sevencop/pow]'` would insert it under `order_field` ;-)